azerothcore/azerothcore-wotlk
# fix(DB/Spell): remove NPC curse

I don't know if my fix is an hackfix or not, please have a look.

## Changes Proposed:
-  remove all auras before updating entry, this will refresh the aura of the NPC

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4476
- Closes https://github.com/chromiecraft/chromiecraft/issues/52


## Tests Performed:
- tested in-game


## How to Test the Changes:
```
.q add 6124
.go cr id 12298
use the item of the quest to heal the deer
it should not have the curse anymore
```


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
